lib/gateway/Shard.js

```diff
@@ -916,12 +916,9 @@ class Shard extends EventEmitter {
                 if(channel) {
                     message = channel.messages.get(packet.d.message_id);
                     if(channel.guild) {
-                        member = channel.guild.members.get(packet.d.user_id);
-                        if(!member && packet.d.member) {
-                            // Updates the member cache with this member for future events.
-                            packet.d.member.id = packet.d.user_id;
-                            member = channel.guild.members.add(packet.d.member, channel.guild);
-                        }
+                        // Updates the member cache with this member for future events.
+                        packet.d.member.id = packet.d.user_id;
```

Comment:
Think the `packet.d.member` guard is still needed, unless it's always present (intents)